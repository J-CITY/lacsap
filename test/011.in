
program HelloWorld;
var
    _int, lazy_town_ : integer;
    __r : real;
Begin
  for i:=2 to n do
  begin
    p:=p*i;
  end;
  int := 1;
END.
