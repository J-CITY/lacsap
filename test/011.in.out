ERROR: ERROR IT IS NOT NUMBER (-1004) row: 0 col: 0
